metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala

```diff
@@ -107,7 +108,26 @@ object ServerCommands {
         |   }
         |}
         |```
-    """.stripMargin
+        |
+        |or DebugUnresolvedMainClassParams object
+        |Example:
+        |```json
+        |{
+        |   mainClass: "com.foo.App",
+        |   buildTarget: "foo",
+        |   args: ["bar"]
```

Comment:
Are we able to also add environment variables? Also, let's add `jvmOptions` to the example.

Reply:
the input for this `Params` class is based on `ScalaMainClass` params, and it does not take environment variables.
About jvmOpts -- can you propose one?

Reply:
@marek1840 Would make sense to add them to BSP? 
@alekseiAlefirov  Maybe `-Dfile.encoding=UTF-8`?

Reply:
added